VoxelSniperCore/src/main/java/com/github/kevindagame/brush/SplatterOverlayBrush.java

```diff
@@ -184,10 +184,8 @@ public HashMap<String, List<String>> registerArgumentValues() {
         // Number variables
         argumentValues.put("recursion", Lists.newArrayList("[number]"));
         argumentValues.put("depth", Lists.newArrayList("[number]"));
-
-        // Decimal variables
-        argumentValues.put("seed", Lists.newArrayList("[decimal]"));
-        argumentValues.put("growth", Lists.newArrayList("[decimal]"));
+        argumentValues.put("seed", Lists.newArrayList("[number]"));
+        argumentValues.put("growth", Lists.newArrayList("[number]"));
```

Comment:
I think most of these are already added by the _SplatterBrushBase_ class (also applies to registerArguments).

Reply:
I realized, but decided to not change since it's outside the scope of this pr. It works as is.

Reply:
I only changed it in splatterball since it was the same exact method